tests/functionaltests/suites/repofilter/post/GetRecords-or1.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
+<csw:GetRecords xmlns:csw="http://www.opengis.net/cat/csw/2.0.2" xmlns:ogc="http://www.opengis.net/ogc" service="CSW" version="2.0.2" resultType="results" startPosition="1" maxRecords="5" outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/cat/csw/2.0.2 http://schemas.opengis.net/csw/2.0.2/CSW-discovery.xsd">
+        <csw:Query typeNames="csw:Record">
```

Comment:
Please update the https://github.com/MapColonies/pycsw/blob/master/tests/functionaltests/suites/repofilter/default.cfg file to include complex "and" and "or" statements inside its "filter=" field to check all cases are covered.